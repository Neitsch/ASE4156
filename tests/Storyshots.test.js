/* eslint-disable */
import initStoryshots from '@storybook/addon-storyshots';

console.log = jest.fn();

initStoryshots({
  storyKindRegex:/^((?!.*?DontTest).)*$/
});
/* eslint-enable */
