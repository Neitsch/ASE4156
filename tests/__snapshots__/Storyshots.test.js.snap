// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots AppBar Main app bar 1`] = `
<div
  style={
    Object {
      "background": "#ffffff",
      "bottom": 0,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    className="SplitPane  vertical "
    style={
      Object {
        "MozUserSelect": "text",
        "WebkitUserSelect": "text",
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "height": "100%",
        "left": 0,
        "msUserSelect": "text",
        "outline": "none",
        "overflow": "hidden",
        "position": "absolute",
        "right": 0,
        "userSelect": "text",
      }
    }
  >
    <div
      className="Pane vertical Pane1"
      style={
        Object {
          "flex": 1,
          "outline": "none",
          "overflowX": "auto",
          "overflowY": "auto",
          "position": "relative",
        }
      }
    >
      <div>
        <div
          className={undefined}
          style={
            Object {
              "WebkitTapHighlightColor": "rgba(0,0,0,0)",
              "backgroundColor": "#CD1309",
              "borderRadius": "0px",
              "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
         0 1px 4px rgba(0, 0, 0, 0.12)",
              "boxSizing": "border-box",
              "color": "rgba(0, 0, 0, 0.87)",
              "display": "flex",
              "fontFamily": "Roboto, sans-serif",
              "muiPrepared": true,
              "paddingLeft": 24,
              "paddingRight": 24,
              "position": "relative",
              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "width": "100%",
              "zIndex": 1100,
            }
          }
        >
          <button
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseOut={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "background": "none",
                "border": 10,
                "boxSizing": "border-box",
                "cursor": "pointer",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 0,
                "fontWeight": "inherit",
                "height": 48,
                "margin": 0,
                "marginLeft": -16,
                "marginRight": 8,
                "marginTop": 8,
                "muiPrepared": true,
                "outline": "none",
                "overflow": "visible",
                "padding": 12,
                "position": "relative",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "verticalAlign": null,
                "width": 48,
                "zIndex": 1,
              }
            }
            tabIndex={0}
            type="button"
          >
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
            >
              <svg
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
                style={
                  Object {
                    "color": "#FFFF00",
                    "display": "inline-block",
                    "fill": "#FFFF00",
                    "height": 24,
                    "muiPrepared": true,
                    "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                    "userSelect": "none",
                    "width": 24,
                  }
                }
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
                />
              </svg>
            </div>
          </button>
          <h1
            onClick={[Function]}
            style={
              Object {
                "boxFlex": 1,
                "color": "#FFFF00",
                "flex": "1",
                "fontSize": 24,
                "fontWeight": 400,
                "height": 64,
                "letterSpacing": 0,
                "lineHeight": "64px",
                "margin": 0,
                "muiPrepared": true,
                "overflow": "hidden",
                "paddingTop": 0,
                "textOverflow": "ellipsis",
                "whiteSpace": "nowrap",
              }
            }
          >
            Wells Fargo
          </h1>
          <button
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseOut={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "background": "none",
                "border": 10,
                "boxSizing": "border-box",
                "cursor": "pointer",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": 0,
                "fontWeight": "inherit",
                "height": 48,
                "margin": 0,
                "marginLeft": "auto",
                "marginRight": -16,
                "marginTop": 8,
                "muiPrepared": true,
                "outline": "none",
                "overflow": "visible",
                "padding": 12,
                "position": "relative",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "verticalAlign": null,
                "width": 48,
                "zIndex": 1,
              }
            }
            tabIndex={0}
            type="button"
          >
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
            >
              <span
                className="muidocs-icon-navigation-expand-more"
                color="rgba(0, 0, 0, 0.87)"
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
                style={
                  Object {
                    "color": "#FFFF00",
                    "display": "inline-block",
                    "fill": "#FFFF00",
                    "fontSize": 24,
                    "muiPrepared": true,
                    "position": "relative",
                    "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                    "userSelect": "none",
                  }
                }
              />
            </div>
          </button>
        </div>
      </div>
    </div>
    <span
      className="Resizer vertical "
      onClick={[Function]}
      onDoubleClick={[Function]}
      onMouseDown={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "cursor": "col-resize",
          "marginRight": -6,
          "width": 10,
          "zIndex": 1,
        }
      }
    />
    <div
      className="Pane vertical Pane2"
      style={
        Object {
          "flex": "none",
          "outline": "none",
          "position": "relative",
          "width": 0,
        }
      }
    >
      <div
        className="sb-addon-material-ui-theme_sidebar"
        style={
          Object {
            "height": "100%",
            "width": "100%",
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`Storyshots Cards Content card 1`] = `
<div
  style={
    Object {
      "background": "#ffffff",
      "bottom": 0,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    className="SplitPane  vertical "
    style={
      Object {
        "MozUserSelect": "text",
        "WebkitUserSelect": "text",
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "height": "100%",
        "left": 0,
        "msUserSelect": "text",
        "outline": "none",
        "overflow": "hidden",
        "position": "absolute",
        "right": 0,
        "userSelect": "text",
      }
    }
  >
    <div
      className="Pane vertical Pane1"
      style={
        Object {
          "flex": 1,
          "outline": "none",
          "overflowX": "auto",
          "overflowY": "auto",
          "position": "relative",
        }
      }
    >
      <div>
        <div
          style={
            Object {
              "WebkitTapHighlightColor": "rgba(0,0,0,0)",
              "backgroundColor": "#ffffff",
              "borderRadius": 2,
              "boxShadow": "0 10px 30px rgba(0, 0, 0, 0.19),
         0 6px 10px rgba(0, 0, 0, 0.23)",
              "boxSizing": "border-box",
              "color": "rgba(0, 0, 0, 0.87)",
              "fontFamily": "Roboto, sans-serif",
              "muiPrepared": true,
              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "zIndex": 1,
            }
          }
        >
          <div
            style={
              Object {
                "paddingBottom": 0,
              }
            }
          >
            <div>
              <h1>
                 Content 
              </h1>
            </div>
          </div>
        </div>
      </div>
    </div>
    <span
      className="Resizer vertical "
      onClick={[Function]}
      onDoubleClick={[Function]}
      onMouseDown={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "cursor": "col-resize",
          "marginRight": -6,
          "width": 10,
          "zIndex": 1,
        }
      }
    />
    <div
      className="Pane vertical Pane2"
      style={
        Object {
          "flex": "none",
          "outline": "none",
          "position": "relative",
          "width": 0,
        }
      }
    >
      <div
        className="sb-addon-material-ui-theme_sidebar"
        style={
          Object {
            "height": "100%",
            "width": "100%",
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`Storyshots Cards Simple card 1`] = `
<div
  style={
    Object {
      "background": "#ffffff",
      "bottom": 0,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    className="SplitPane  vertical "
    style={
      Object {
        "MozUserSelect": "text",
        "WebkitUserSelect": "text",
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "height": "100%",
        "left": 0,
        "msUserSelect": "text",
        "outline": "none",
        "overflow": "hidden",
        "position": "absolute",
        "right": 0,
        "userSelect": "text",
      }
    }
  >
    <div
      className="Pane vertical Pane1"
      style={
        Object {
          "flex": 1,
          "outline": "none",
          "overflowX": "auto",
          "overflowY": "auto",
          "position": "relative",
        }
      }
    >
      <div>
        <div
          style={
            Object {
              "WebkitTapHighlightColor": "rgba(0,0,0,0)",
              "backgroundColor": "#ffffff",
              "borderRadius": 2,
              "boxShadow": "0 10px 30px rgba(0, 0, 0, 0.19),
         0 6px 10px rgba(0, 0, 0, 0.23)",
              "boxSizing": "border-box",
              "color": "rgba(0, 0, 0, 0.87)",
              "fontFamily": "Roboto, sans-serif",
              "muiPrepared": true,
              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "zIndex": 1,
            }
          }
        >
          <div
            style={
              Object {
                "paddingBottom": 8,
              }
            }
          >
            <div
              style={
                Object {
                  "muiPrepared": true,
                  "padding": 16,
                  "position": "relative",
                }
              }
            >
              <span
                style={
                  Object {
                    "color": "rgba(0, 0, 0, 0.87)",
                    "display": "block",
                    "fontSize": 24,
                    "lineHeight": "36px",
                    "muiPrepared": true,
                  }
                }
              >
                Title
              </span>
              <span
                style={
                  Object {
                    "color": "rgba(0, 0, 0, 0.54)",
                    "display": "block",
                    "fontSize": 14,
                    "muiPrepared": true,
                  }
                }
              >
                sub
              </span>
            </div>
            <div
              style={
                Object {
                  "color": "rgba(0, 0, 0, 0.87)",
                  "fontSize": 14,
                  "muiPrepared": true,
                  "padding": 16,
                }
              }
            >
              fdsa
            </div>
          </div>
        </div>
      </div>
    </div>
    <span
      className="Resizer vertical "
      onClick={[Function]}
      onDoubleClick={[Function]}
      onMouseDown={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "cursor": "col-resize",
          "marginRight": -6,
          "width": 10,
          "zIndex": 1,
        }
      }
    />
    <div
      className="Pane vertical Pane2"
      style={
        Object {
          "flex": "none",
          "outline": "none",
          "position": "relative",
          "width": 0,
        }
      }
    >
      <div
        className="sb-addon-material-ui-theme_sidebar"
        style={
          Object {
            "height": "100%",
            "width": "100%",
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`Storyshots Saldo Playground 1`] = `
<div
  style={
    Object {
      "background": "#ffffff",
      "bottom": 0,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    className="SplitPane  vertical "
    style={
      Object {
        "MozUserSelect": "text",
        "WebkitUserSelect": "text",
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "height": "100%",
        "left": 0,
        "msUserSelect": "text",
        "outline": "none",
        "overflow": "hidden",
        "position": "absolute",
        "right": 0,
        "userSelect": "text",
      }
    }
  >
    <div
      className="Pane vertical Pane1"
      style={
        Object {
          "flex": 1,
          "outline": "none",
          "overflowX": "auto",
          "overflowY": "auto",
          "position": "relative",
        }
      }
    >
      <div>
        <div
          style={
            Object {
              "WebkitTapHighlightColor": "rgba(0,0,0,0)",
              "backgroundColor": "#ffffff",
              "borderRadius": 2,
              "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
         0 1px 4px rgba(0, 0, 0, 0.12)",
              "boxSizing": "border-box",
              "color": "rgba(0, 0, 0, 0.87)",
              "fontFamily": "Roboto, sans-serif",
              "muiPrepared": true,
              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "zIndex": 1,
            }
          }
        >
          <div
            style={
              Object {
                "paddingBottom": 8,
              }
            }
          >
            <div
              style={
                Object {
                  "boxSizing": "border-box",
                  "fontWeight": 500,
                  "muiPrepared": true,
                  "padding": 16,
                  "position": "relative",
                  "whiteSpace": "nowrap",
                }
              }
            >
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "muiPrepared": true,
                    "paddingRight": "90px",
                    "verticalAlign": "top",
                    "whiteSpace": "normal",
                  }
                }
              >
                <span
                  style={
                    Object {
                      "color": "rgba(0, 0, 0, 0.87)",
                      "display": "block",
                      "fontSize": 15,
                      "muiPrepared": true,
                    }
                  }
                >
                  Saldo
                </span>
                <span
                  style={
                    Object {
                      "color": "rgba(0, 0, 0, 0.54)",
                      "display": "block",
                      "fontSize": 14,
                      "muiPrepared": true,
                    }
                  }
                />
              </div>
            </div>
            <div
              style={
                Object {
                  "color": "rgba(0, 0, 0, 0.87)",
                  "fontSize": 14,
                  "muiPrepared": true,
                  "padding": 16,
                }
              }
            >
              <div
                style={
                  Object {
                    "height": "auto",
                    "muiPrepared": true,
                    "overflow": "auto",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "muiPrepared": true,
                    }
                  }
                >
                  <table
                    className={undefined}
                    style={
                      Object {
                        "backgroundColor": "#ffffff",
                        "borderCollapse": "collapse",
                        "borderSpacing": 0,
                        "fontFamily": "Roboto, sans-serif",
                        "tableLayout": "fixed",
                        "width": "100%",
                      }
                    }
                  />
                </div>
                <div
                  style={
                    Object {
                      "height": "inherit",
                      "muiPrepared": true,
                      "overflowX": "hidden",
                      "overflowY": "auto",
                    }
                  }
                >
                  <table
                    className={undefined}
                    style={
                      Object {
                        "backgroundColor": "#ffffff",
                        "borderCollapse": "collapse",
                        "borderSpacing": 0,
                        "fontFamily": "Roboto, sans-serif",
                        "tableLayout": "fixed",
                        "width": "100%",
                      }
                    }
                  >
                    <tbody
                      style={
                        Object {
                          "muiPrepared": true,
                        }
                      }
                    >
                      <tr
                        className={undefined}
                        style={
                          Object {
                            "borderBottom": false,
                            "color": "rgba(0, 0, 0, 0.87)",
                            "height": 48,
                            "muiPrepared": true,
                          }
                        }
                      >
                        <td
                          className={undefined}
                          onClick={[Function]}
                          onMouseEnter={[Function]}
                          onMouseLeave={[Function]}
                          style={
                            Object {
                              "backgroundColor": "inherit",
                              "fontSize": 13,
                              "height": 48,
                              "muiPrepared": true,
                              "overflow": "hidden",
                              "paddingLeft": 24,
                              "paddingRight": 24,
                              "textAlign": "left",
                              "textOverflow": "ellipsis",
                              "whiteSpace": "nowrap",
                            }
                          }
                        >
                          investedSharesValue
                        </td>
                        <td
                          className={undefined}
                          onClick={[Function]}
                          onMouseEnter={[Function]}
                          onMouseLeave={[Function]}
                          style={
                            Object {
                              "backgroundColor": "inherit",
                              "fontSize": 13,
                              "height": 48,
                              "muiPrepared": true,
                              "overflow": "hidden",
                              "paddingLeft": 24,
                              "paddingRight": 24,
                              "textAlign": "right",
                              "textOverflow": "ellipsis",
                              "whiteSpace": "nowrap",
                            }
                          }
                        >
                          $700.00
                        </td>
                      </tr>
                      <tr
                        className={undefined}
                        style={
                          Object {
                            "borderBottom": false,
                            "color": "rgba(0, 0, 0, 0.87)",
                            "height": 48,
                            "muiPrepared": true,
                          }
                        }
                      >
                        <td
                          className={undefined}
                          onClick={[Function]}
                          onMouseEnter={[Function]}
                          onMouseLeave={[Function]}
                          style={
                            Object {
                              "backgroundColor": "inherit",
                              "fontSize": 13,
                              "height": 48,
                              "muiPrepared": true,
                              "overflow": "hidden",
                              "paddingLeft": 24,
                              "paddingRight": 24,
                              "textAlign": "left",
                              "textOverflow": "ellipsis",
                              "whiteSpace": "nowrap",
                            }
                          }
                        >
                          earnedSharesValue
                        </td>
                        <td
                          className={undefined}
                          onClick={[Function]}
                          onMouseEnter={[Function]}
                          onMouseLeave={[Function]}
                          style={
                            Object {
                              "backgroundColor": "inherit",
                              "fontSize": 13,
                              "height": 48,
                              "muiPrepared": true,
                              "overflow": "hidden",
                              "paddingLeft": 24,
                              "paddingRight": 24,
                              "textAlign": "right",
                              "textOverflow": "ellipsis",
                              "whiteSpace": "nowrap",
                            }
                          }
                        >
                          $74.93
                        </td>
                      </tr>
                      <tr
                        className={undefined}
                        style={
                          Object {
                            "borderBottom": false,
                            "color": "rgba(0, 0, 0, 0.87)",
                            "height": 48,
                            "muiPrepared": true,
                          }
                        }
                      >
                        <td
                          className={undefined}
                          onClick={[Function]}
                          onMouseEnter={[Function]}
                          onMouseLeave={[Function]}
                          style={
                            Object {
                              "backgroundColor": "inherit",
                              "fontSize": 13,
                              "height": 48,
                              "muiPrepared": true,
                              "overflow": "hidden",
                              "paddingLeft": 24,
                              "paddingRight": 24,
                              "textAlign": "left",
                              "textOverflow": "ellipsis",
                              "whiteSpace": "nowrap",
                            }
                          }
                        >
                          bonusSharesValue
                        </td>
                        <td
                          className={undefined}
                          onClick={[Function]}
                          onMouseEnter={[Function]}
                          onMouseLeave={[Function]}
                          style={
                            Object {
                              "backgroundColor": "inherit",
                              "fontSize": 13,
                              "height": 48,
                              "muiPrepared": true,
                              "overflow": "hidden",
                              "paddingLeft": 24,
                              "paddingRight": 24,
                              "textAlign": "right",
                              "textOverflow": "ellipsis",
                              "whiteSpace": "nowrap",
                            }
                          }
                        >
                          $0.00
                        </td>
                      </tr>
                      <tr
                        className={undefined}
                        style={
                          Object {
                            "borderBottom": "1px solid #e0e0e0",
                            "color": "rgba(0, 0, 0, 0.87)",
                            "height": 48,
                            "muiPrepared": true,
                          }
                        }
                      >
                        <td
                          className={undefined}
                          onClick={[Function]}
                          onMouseEnter={[Function]}
                          onMouseLeave={[Function]}
                          style={
                            Object {
                              "backgroundColor": "inherit",
                              "fontSize": 13,
                              "height": 48,
                              "muiPrepared": true,
                              "overflow": "hidden",
                              "paddingLeft": 24,
                              "paddingRight": 24,
                              "textAlign": "left",
                              "textOverflow": "ellipsis",
                              "whiteSpace": "nowrap",
                            }
                          }
                        >
                          reinvestedSharesValue
                        </td>
                        <td
                          className={undefined}
                          onClick={[Function]}
                          onMouseEnter={[Function]}
                          onMouseLeave={[Function]}
                          style={
                            Object {
                              "backgroundColor": "inherit",
                              "fontSize": 13,
                              "height": 48,
                              "muiPrepared": true,
                              "overflow": "hidden",
                              "paddingLeft": 24,
                              "paddingRight": 24,
                              "textAlign": "right",
                              "textOverflow": "ellipsis",
                              "whiteSpace": "nowrap",
                            }
                          }
                        >
                          $0.00
                        </td>
                      </tr>
                      <tr
                        className={undefined}
                        style={
                          Object {
                            "borderBottom": false,
                            "color": "rgba(0, 0, 0, 0.87)",
                            "height": 48,
                            "muiPrepared": true,
                          }
                        }
                      >
                        <td
                          className={undefined}
                          onClick={[Function]}
                          onMouseEnter={[Function]}
                          onMouseLeave={[Function]}
                          style={
                            Object {
                              "backgroundColor": "inherit",
                              "fontSize": 13,
                              "height": 48,
                              "muiPrepared": true,
                              "overflow": "hidden",
                              "paddingLeft": 24,
                              "paddingRight": 24,
                              "textAlign": "left",
                              "textOverflow": "ellipsis",
                              "whiteSpace": "nowrap",
                            }
                          }
                        >
                          totalSharesValue
                        </td>
                        <td
                          className={undefined}
                          onClick={[Function]}
                          onMouseEnter={[Function]}
                          onMouseLeave={[Function]}
                          style={
                            Object {
                              "backgroundColor": "inherit",
                              "fontSize": 13,
                              "height": 48,
                              "muiPrepared": true,
                              "overflow": "hidden",
                              "paddingLeft": 24,
                              "paddingRight": 24,
                              "textAlign": "right",
                              "textOverflow": "ellipsis",
                              "whiteSpace": "nowrap",
                            }
                          }
                        >
                          $774.93
                        </td>
                      </tr>
                    </tbody>
                  </table>
                </div>
              </div>
              <button
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                    "backgroundColor": "#a4c639",
                    "border": 10,
                    "borderRadius": 2,
                    "boxSizing": "border-box",
                    "color": "rgba(0, 0, 0, 0.87)",
                    "cursor": "pointer",
                    "display": "inline-block",
                    "fontFamily": "Roboto, sans-serif",
                    "fontSize": "inherit",
                    "fontWeight": "inherit",
                    "height": 36,
                    "lineHeight": "36px",
                    "margin": 0,
                    "minWidth": "100%",
                    "muiPrepared": true,
                    "outline": "none",
                    "overflow": "hidden",
                    "padding": 0,
                    "position": "relative",
                    "textAlign": "center",
                    "textDecoration": "none",
                    "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                    "userSelect": "none",
                    "verticalAlign": null,
                    "zIndex": 1,
                  }
                }
                tabIndex={0}
                type="button"
              >
                <div
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                >
                  <span
                    style={
                      Object {
                        "fontSize": 14,
                        "fontWeight": 500,
                        "letterSpacing": 0,
                        "muiPrepared": true,
                        "paddingLeft": 16,
                        "paddingRight": 16,
                        "position": "relative",
                        "textTransform": "uppercase",
                        "verticalAlign": "middle",
                      }
                    }
                  >
                    Sell Shares
                  </span>
                </div>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <span
      className="Resizer vertical "
      onClick={[Function]}
      onDoubleClick={[Function]}
      onMouseDown={[Function]}
      onTouchEnd={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "cursor": "col-resize",
          "marginRight": -6,
          "width": 10,
          "zIndex": 1,
        }
      }
    />
    <div
      className="Pane vertical Pane2"
      style={
        Object {
          "flex": "none",
          "outline": "none",
          "position": "relative",
          "width": 0,
        }
      }
    >
      <div
        className="sb-addon-material-ui-theme_sidebar"
        style={
          Object {
            "height": "100%",
            "width": "100%",
          }
        }
      />
    </div>
  </div>
</div>
`;
